tests/default.nf.test

```diff
@@ -15,22 +15,113 @@ nextflow_pipeline {
+
+            //kmcp
+            def stable_name_kmcp         = getAllFilesFromDir("$outputDir/kmcp", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            // krona
+            def stable_name_krona        = getAllFilesFromDir("$outputDir/krona", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            // metaphlan
+            def stable_name_metaphlan    = getAllFilesFromDir("$outputDir/metaphlan", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            // nanoq
+            def stable_name_nanoq        = getAllFilesFromDir("$outputDir/nanoq", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            // porechop_abi
+            def stable_content_porechopabi = getAllFilesFromDir("$outputDir/porechop_abi", relative: true, includeDir: false, ignore: ['*.{log}'] , ignoreFile: null, include: ['*','**/*'])
+            def stable_name_porechopabi  = getAllFilesFromDir("$outputDir/porechop_abi", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*.log'])
+
+            // samtools
+            def stable_name_samtools     = getAllFilesFromDir("$outputDir/samtools", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+            //taxpasta
+            def stable_name_taxpasta     = getAllFilesFromDir("$outputDir/taxpasta", relative: true, includeDir: false, ignore: null, ignoreFile: null, include: ['*', '**/*'])
+
+
             assertAll(
                 { assert workflow.success},
                 { assert snapshot(
                     // Number of successful tasks
                     workflow.trace.succeeded().size(),
                     // pipeline versions.yml file for multiqc from which Nextflow version is removed because we test pipelines on multiple Nextflow versions
                     removeNextflowVersion("$outputDir/pipeline_info/nf_core_taxprofiler_software_mqc_versions.yml"),
-                    // All stable path name, with a relative path
-                   // stable_name,
-                    // All files with stable contents
-                    //stable_path
-                ).match() }
+                ).match() },
+
+                { assert new File("$outputDir/pipeline_info/nf_core_taxprofiler_software_mqc_versions.yml").exists() },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+
+                { assert snapshot ( stable_name_all ).match("all_files") },
+		        { assert snapshot ( stable_name_multiqc ).match("multiqc" ) },
+                { assert snapshot( file("$outputDir/bbduk/2613_ERR5766181.bbduk.log").text.contains("Reads Processed:"), [stable_content_bbduk, stable_name_bbduk]).match("bbduk")},
```

Comment:
Why have you put content/name as a list here?, they can be independent entries

Reply:
I will do so!